lib/rucio/api/rule.py

```diff
@@ -108,7 +108,7 @@ def get_replication_rule(rule_id, issuer, estimate_ttc=None, vo='def'):
     :param vo: The VO of the issuer.
     """
     kwargs = {'rule_id': rule_id}
-    if not has_permission(issuer=issuer, vo=vo, action='access_rule', kwargs=kwargs):
+    if not has_permission(issuer=issuer, vo=vo, action='access_rule_vo', kwargs=kwargs):
         raise AccessDenied('Account %s can not access rules at other VOs.' % (issuer))
```

Comment:
I know Martin said we should avoid Exceptions that imply the existence of other VOs, but I suppose this is OK because we'd trigger this by specifying the rule_id rather than the VO (so it won't expose anything about the VO to which the rule belongs other than the fact it exists)?

Reply:
I messaged Martin about this and he concluded that it's all right to leave the message as it is in this case - particularly as it can only be triggered by a mismatch in the rule and issuer VOs